bundle/EventListener/Shop/ProductIndexListener.php

```diff
@@ -13,6 +13,7 @@
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\RequestStack;
 
+/** @deprecated since 1.4.6 together with nglayouts_sylius_taxon attribute, use nglayouts_sylius_resource instead */
```

Comment:
Maybe it would be better to link to other event listener instead of just mentioning the new key. @emodric WDYT?

Reply:
Sure, why not. We can mention it by name: `use nglayouts_sylius_resource attribute set by BlaBla listener instead`

Reply:
Also, please expand the comment to multiline:

```
/**
 * @deprecated ...
 */
```